cypress/tsconfig.json

```diff
@@ -8,5 +8,5 @@
       "@types/cypress-image-snapshot"
     ]
   },
-  "include": ["*", "specs"]
+  "include": ["*", "e2e"]
```

Comment:
I switched to the default folder (it used to be `integration` which I didn't like) to avoid having to specify a regex in the config.